app/controllers/editorial/news_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Editorial
```

Comment:
Time investment at the moment - there's some custom stuff we want to handle with the routes (don't offer up a choice for content types when creating a new one etc.) that means this is the quickest path forward until we have a few more cases like this to identify the pattern.

I suspect it will be possible to achieve this with our node objects and routing, but we'll need to build in features for sections to restrict content types, visibility and access to special cases (i.e. no one is really restricted to news, but they are restricted to the publishing section/s).